lib/src/main/java/com/ragnarok/rxcamera/RxCameraInternal.java

```diff
@@ -381,9 +385,10 @@ public boolean closeCameraInternal() {
     }
 
     @Override
-    public void onAvailable() {
+    public void onAvailable(SurfaceHolder holder) {
         if (isNeedStartPreviewLater) {
             try {
+                camera.setPreviewDisplay(holder);
```

Comment:
you forgot to add the previewDisplay when the Surface was created, or the Texture was available, this is important if you don't have an open button, and you want to preview as soon as the surface/texture is available